.gitlab/pipeline/short-benchmarks.yml

```diff
@@ -24,3 +27,58 @@ short-benchmark-westend:
   <<: *short-bench
   variables:
     RUNTIME: westend
+
+# run short-benchmarks for system parachain runtimes from cumulus
+
+.short-benchmark-cumulus: &short-bench-cumulus
+  stage: short-benchmarks
+  extends:
+    - .common-refs
+    - .docker-env
+  needs:
+    - job: build-short-benchmark-cumulus
+      artifacts: true
+  variables:
+    RUNTIME_CHAIN: benchmarked-runtime-chain
```

Comment:
```suggestion
    RUNTIME_CHAIN: benchmarked-runtime-chain
  tags:
    - benchmark
```

Reply:
@alvicsam ok, but I dont know how, can you help with that?
or that suggested change is the fix?

Reply:
yep, the suggested fix will do that

Reply:
@alvicsam do we need to add that tag also to [short-benchmark-polkadot](https://github.com/paritytech/polkadot-sdk/pull/1183/files#diff-37ac91a4b108440e9511bd13341ff237deebb3cf99815466cafe8b4468939e7bR8-R17)?

Reply:
ok, fixed

Reply:
thank you!